src/polyfill.js

```diff
@@ -22,13 +22,13 @@ function setup() {
     if (typeof window[GLOBALNAME] === "function") {
         return;
     }
-    //
     const sanitizer = function(config) {
       const api = Object.create({});
 
       let normalizedConfig = _normalizeConfig(config);
       Object.assign(api, {
         sanitize(input) {
+          // Q: _fragmentParser accepts `input` as type DOMString. We might need to update the code block below 
```

Comment:
The universal usage of `input` all over the place is probably a mistake :)
Also, declaring the function inline like here makes us easier to omit documentation in jsdoc, which will turn this even more error-prone.

Maybe we'll leave `sanitizer` as it is right now and get back to it later. Please remove the comment.

Reply:
Ohh, whoops! I think I see what you mean - the `input` parameter in `sanitize` is a different `input` parameter than the one for `sanitizeFor` 

> Also, declaring the function inline like here makes us easier to omit documentation in jsdoc, which will turn this even more error-prone.

Ah, I ran into this exact issue with https://github.com/mozilla/sanitizer-polyfill/pull/8#issuecomment-875216776

> Maybe we'll leave sanitizer as it is right now and get back to it later. Please remove the comment.

Sounds good! I'll remove the comment